lib/devices/ios/ios-controller.js

```diff
@@ -41,6 +44,13 @@ var logTypesSupported = {
   'performance': 'Performance Logs - Debug Timelines on real devices and simulators'
 };
 
+var getDefaultCallbackAddress = function () {
+  var parser = new Parser();
+  return parser.rawArgs.filter(function (a) {
+      return a[0][1] === '--address';
```

Comment:
these two lines are indented too far.

also, is Array.filter implemented in node 0.10?